### PR TITLE
Forum + Comments  MVP

### PR DESCRIPTION
**Pull request by jasonify**

### CODE REVIEW

**Comment by TheHollidayInn** [comment]
Sweet! I'll review this during the week. 2 notes immediately:
- You still have the recommended links in here. In the future, I recommend starting straight off develop each time. Or merging in to old PRs
- Small commits are good, but you have quite a few making it hard to follow the changes based on them.

**Comment by jasonify** [comment]
Agreed, let me know any other thoughts. 

- I do think it would be good to merge in the comment and reply cell into one common component like Justin had suggested.  At the very least make an issue for this.
- I think upvote API call could be one call for both forum + posts.
- The ForumPostLite and PostLite models could be removed by modifying the originals to accept a string or an object for certain properties 

Missing:
-  Sadly there is no compose forum thread for this MVP.  Should be pretty easy to add.
-  Mentions not yet supported 
- inline links don't get detected by the markdown library I am using.
